sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/ContainerRegistryAudience.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.util.Collection;
+
+/** Defines values for ContainerRegistryAudience. */
+public class ContainerRegistryAudience extends ExpandableStringEnum<ContainerRegistryAudience> {
+    /** Static value AzureResourceManagerChina for ContainerRegistryAudience. */
+    public static final ContainerRegistryAudience AZURERESOURCEMANAGERCHINA = fromString("https://management.chinacloudapi.cn");
```

Comment:
Should this live somewhere else given that this seems to be generic, and may be applicable for many more SDKs

Reply:
We ultimately want to remove/hide the need for these fields in ACR. ACR uses control plane scopes in its data plane library and will eventually change it to use other scopes.

So, in future there will be other fields added to the ContainerRegistryAudience type which will be specific to ACR.